client/src/views/Simulation/components/VariablesPanel.vue

```diff
@@ -164,20 +159,40 @@
     @Action hideAllVariables;
     @Action showAllVariables;
 
+    calcPolygon (variable: HMI.SimulationVariable): void {
+      const numRuns = variable.values.length;
+      const xArr = new Array(numRuns - 1);
+      const yMinArr = new Array(numRuns - 1);
+      const yMaxArr = new Array(numRuns - 1);
+
+      for (let runIndex = 0; runIndex < numRuns - 1; runIndex++) {
+        const run = variable.values.shift();
+        run.forEach((point, pointIndex) => {
+          xArr[pointIndex] = point.x;
+          yMinArr[pointIndex] = Math.min(yMinArr[pointIndex] ?? point.y, point.y);
+          yMaxArr[pointIndex] = Math.max(yMaxArr[pointIndex] ?? point.y, point.y);
+        });
+      }
+
+      variable.polygon = [
+        ...yMinArr.map((y, i) => ({ x: xArr[i], y })),
+        ...yMaxArr.map((y, i) => ({ x: xArr[i], y })).reverse(),
```

Comment:
Could you add a small comment explaining why you use `reverse()`, it's not obvious right away.